src/test/java/org/llorllale/cactoos/matchers/TextHasStringTest.java

```diff
@@ -57,8 +57,7 @@ public void hasClearDescriptionForFailedTest() throws Exception {
         final TextHasString matcher = new TextHasString(
             "ed076287532e86365e841e92bfc50d8c6"
         );
-        matcher.matchesSafely(hex);
-        matcher.describeMismatchSafely(hex, description);
+        matcher.matchesSafely(hex, description);
```

Comment:
@dgroup why this change?

Reply:
Because now the TestHasString is extending TypeSafeDiagnosticMatcher instead of TypeSafeMatcher.